SU-SOE/soe_basic
# D8CORE-3960: fixin' the publications read more

# READY FOR REVIEW

# Summary
- Fixing the display of the Read more publications cards at the foot of a publication node.
- added the SOE arrow to the link
- corrected the more-link button

# Review By (Date)
- 3/26

# Urgency
- Med

# Steps to Test

1. Pull in this change.
2. Create 4 publications.
3. Look at the cards on the node.
4. Verify they match the Design in the Jira. 
5. Verify the edit publication button on the view looks correct.

# Affected Projects or Products
- SOE Theme

# Associated Issues and/or People
- D8CORE-3960
- https://github.com/SU-SWS/stanford_publication/pull/32 - A publications module pull request. It isn't dependent but probably better to test together. 
 
# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
